chrome/content/zotero/zoteroPane.js

```diff
@@ -4091,13 +4091,38 @@ var ZoteroPane = new function()
 
 	this.buildCopyCollectionMenu = function (event) {
 		if (event.target !== event.currentTarget) return;
-		let popup = event.target;
+		let popup = document.getElementById("zotero-copy-collection-popup");
 		popup.replaceChildren();
 		let selected = this.getSelectedCollection();
 
 		// Fetch all libraries
 		let topLevelEntries = Zotero.Libraries.getAll().filter(lib => !(lib instanceof Zotero.Feed));
 
+		// Check which libraries have collections linked to the selected collection
+		// and disable their menuitems. Same logic as in CollectionTree.canDropCheckAsync.
+		let linkedCollectionsExist = {};
+		(async () => {
+			for (let library of Zotero.Libraries.getAll()) {
+				if (library.libraryID == selected.libraryID || library instanceof Zotero.Feed) continue;
+				// Check which library has a collection linked to the selected collection
+				let linkedCollection = await selected.getLinkedCollection(library.libraryID, true);
+				linkedCollectionsExist[library.libraryID] = linkedCollection;
+				// Also check which library has collections linked to a subcollection of the selected collection
+				for (let descendent of selected.getDescendents(false, 'collection')) {
+					let subcollection = Zotero.Collections.get(descendent.id);
+					let linkedSubcollection = await subcollection.getLinkedCollection(library.libraryID, true);
+					linkedCollectionsExist[library.libraryID] = linkedSubcollection || linkedCollectionsExist[library.libraryID];
+				}
+			}
+			// Libraries that have linked collections have their menus disabled
+			for (let libraryMenuItem of [...popup.childNodes]) {
+				let menuItemLibID = libraryMenuItem.getAttribute("value").substring(1);
+				if (linkedCollectionsExist[parseInt(menuItemLibID)]) {
```

Comment:
No real reason to use `parseInt` here as is, since the object key is always a string anyway. (Or you could use `Map` instead of an array for `linkedCollectionsExist`, and then they'd be actual numbers.)